nekoyume/Assets/_Scripts/UI/Widget/Popup/CostTwoButtonPopup.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Nekoyume.Game.Controller;
+using Nekoyume.Model.Item;
+using Nekoyume.State;
+using Nekoyume.UI.Module;
+using TMPro;
+using UnityEngine;
+
+namespace Nekoyume.UI
+{
+    using UniRx;
+    public class CostTwoButtonPopup : PopupWidget
+    {
+        [SerializeField]
+        private ConditionalCostButton costButton = null;
```

Comment:
```suggestion
        private ConditionalCostButton costButton;
```

other `SerializeField` too.